migrations/versions/6904ae1e0e33_.py

```diff
@@ -0,0 +1,49 @@
+"""empty message
+
+Revision ID: 6904ae1e0e33
+Revises: ccd80550c01f
+Create Date: 2017-06-13 08:42:28.606478
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '6904ae1e0e33'
+down_revision = 'ccd80550c01f'
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('image_sizes', 'full_aspect',
```

Comment:
@niranjan94  Please review again. I have checked, all existing values changes to true/false.